README.md

```diff
@@ -46,6 +46,64 @@ Set the USERNAME and PASSWORD as secrets with `npx wrangler secret put USERNAME
 You can add a base64 encoded JWT public key to verify passwords (or token) that are signed by the private key.
 `npx wrangler secret put JWT_REGISTRY_TOKENS_PUBLIC_KEY --env production`
 
+### Using with Docker
+
+You can use this registry with Docker to push and pull images.
+
+Example using `docker push` and `docker pull`:
+
+```bash
+export REGISTRY_URL=your-url-here
+
+# Replace $PASSWORD and $USERNAME with the actual credentials
+echo $PASSWORD > docker login --username $USERNAME --password-stdin $REGISTRY_URL
```

Comment:
echo $PASSWORD | docker login --username $USERNAME --password-stdin $REGISTRY_URL
Pipe is better here?